ci/erlang.yml

```diff
@@ -6,10 +6,14 @@ on:
   pull_request:
     branches: [ $default-branch ]
 
+permissions: read-all
```

Comment:
Why is this needed?

Reply:
This permission at the workflow (top) level is so that any future jobs that get added to the same workflow are secure by default. It also makes the changes in-line with expectation from [ossf/scorecard](https://github.com/ossf/scorecard). You can see the discussion [here](https://github.com/ossf/scorecard/issues/1128#issuecomment-973236716). 

CC: @laurentsimon

Reply:
The discussion ⬆️ was good read. Thanks!
And I agree it makes sense to add permission at workflow level for new jobs.